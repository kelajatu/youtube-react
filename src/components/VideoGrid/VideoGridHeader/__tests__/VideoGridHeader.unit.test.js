import React from "react";
import { shallow } from "enzyme";
import { VideoGridHeader } from "../VideoGridHeader";

describe("VideoGridHeder", () => {
  it("renders without props", () => {
    const wrapper = shallow(<VideoGridHeader />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders with empty string header", () => {
    const wrapper = shallow(<VideoGridHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders with title", () => {
    const wrapper = shallow(<VideoGridHeader title="Autos & Vehicles" />);
    expect(wrapper).toMatchSnapshot();
  });
});
