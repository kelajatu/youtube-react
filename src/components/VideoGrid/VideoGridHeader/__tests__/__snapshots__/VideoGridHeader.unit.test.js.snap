// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeder renders with empty string header 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeder renders with title 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    Autos & Vehicles
  </span>
</div>
`;

exports[`VideoGridHeder renders without props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;
